spec/controllers/course/lesson_plan/items_controller_spec.rb

```diff
@@ -75,6 +75,44 @@
                 to include(I18n.t('components.video.name'))
             end
           end
+
+          context 'when the video component is disabled on the course' do
+            before do
+              instance.settings(:components, :course_videos_component).enabled = true
```

Comment:
There are traits defined in the factory to enable video components for `instance` and `course`. You can use those instead.